src/main/java/servlets/ReceiptAnalysis.java

```diff
@@ -64,6 +64,7 @@ private static AnalysisResults retrieveText(ByteString imageBytes) throws IOExce
     ImmutableList<AnnotateImageRequest> requests = ImmutableList.of(request);
 
     try (ImageAnnotatorClient client = ImageAnnotatorClient.create()) {
+      // TODO: Throw custom exception if response has an error or is missing
```

Comment:
Can you also put the pull request you're waiting for in this TODO. This way someone reading through the code knows why it hasn't been done yet.